cvmfs/s3fanout.h

```diff
@@ -172,8 +178,9 @@ class S3FanoutManager : SingleCopy {
 
   struct S3Config {
     S3Config() {
-      authz_method = kAuthzAwsV2;
+      authz_method = kAuthzAwsV4;
```

Comment:
The deprecation schedule might be different for 3rd party products (MinIO, Ceph, etc). In any case, let's handle the question of the default authz scheme in another PR.  I think we can reuse the `CVMFS_S3_FLAVOR` settings for this instead of guessing based on the region setting.